.github/workflows/clippy.yml

```diff
@@ -0,0 +1,17 @@
+on: pull_request
+
+name: Clippy Check
+jobs:
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          components: clippy
```

Comment:
Do we really need `nightly` for Clippy? I haven't seen that some unstable Clippy features were used. So I suggest to go with `stable` one.

Reply:
I chose nightly's Clippy since it has new lints and the fix for false positives/negatives.

Reply:
But consider that it may accidentally introduce new unstable lints, which doesn't work well. At my job we had quite a pain with that, as CI sporadically failed. If it's not a big problem - I'm OK with that, anyway.

Reply:
`clippy check` only fails when the code has errors (i.e. denied lints detect something) and we can ignore it unless the check is set as required status so it shouldn't be a pain here, I think.